src/components/Trade/Trade.tsx

```diff
@@ -41,7 +46,7 @@ export const Trade = () => {
     <FormProvider {...methods}>
       <MemoryRouter initialEntries={entries}>
         <Switch>
-          <Route path='/' component={TradeRoutes} />
+          <Route path='/' render={() => <TradeRoutes defaultBuyAssetId={defaultBuyAssetId} />} />
```

Comment:
Nitpick: We're not doing anything here that would justify a render method

```suggestion
          <Route path='/'>
            <TradeRoutes defaultBuyAssetId={defaultBuyAssetId} />
          </Route>
```